index.html

```diff
@@ -272,7 +272,7 @@
               horizontal: '-5px',
             },
           },
-          zIndex: 3,
+          zIndex: 1001,

```

Comment:
pls revert 
![image](https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/581d2f7e-e9fa-470b-baaa-d7dcf17cfbfb)

Reply:
![image](https://github.com/KyberNetwork/kyberswap-interface/assets/33005392/eeceb333-d35c-4428-ae51-f9902dac4001)

Reply:
yes, acknowledge this